2021/1001xxx/GSD-2021-1001147.json

```diff
@@ -9,21 +11,30 @@
     "impact": "SQL Injection",
     "credit": "",
     "references": [
-      "https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0017"
+      {
+        "type": "WEB",
+        "url": "https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0017"
+      }
+    ],
+    "affected": [
+      {
+        "package": {
+          "name": "Secure Remote Access (SRA)",
+          "ecosystem": "GSD"
+        },
+        "versions": [
+          "8.x, 9.0.0.9-26sv and earlier"
```

Comment:
This should be multiple strings `["8.x", "9.0.0.9-26sv and earlier"]` as defined [here](https://ossf.github.io/osv-schema/#affectedversions-field). Also It might be helpful to adopt a formal version notation, because the "and ealier" part is very vague.

Reply:
Again this is part of the problem, bad data, and I'm not going to completely cvlean it up one by one, I'm currently getting the files into json compliant state, a data compliant state is step #4.